build.gradle

```
@@ -62,3 +78,86 @@ if (project.hasProperty("teamcity"))
 {
     apply plugin: 'org.labkey.teamCity'
 }
+
+// the jar file that contains just the classes from this project, not all other
+// projects with src/test directories
+project.sourceSets {
+    main {
+        java {
+            srcDirs = ["src"]
+            exclude "test/tests/**"
+        }
+    }
+}
+project.tasks.jar {
+    Jar jar ->
+        jar.archiveBaseName.set(apiArtifactName)
+        jar.setGroup('org.labkey.api')
+}
+
+
+if (project.hasProperty('doPublishing'))
+{
+    project.afterEvaluate {
+        PomFileHelper pomUtil = new PomFileHelper(project)
+        project.publishing {
+            publications {
+                libs(MavenPublication) {
+                    groupId = project.group
+                    artifactId = apiArtifactName
+                    artifact project.tasks.jar
+                    pom {
+                        name = "LabKey Test Automation Classes"
+                        description = "Library of classes and utilities for Selenium testing of LabKey Server components"
+                        url = PomFileHelper.LABKEY_ORG_URL
+                        developers PomFileHelper.getLabKeyTeamDevelopers()
+                        licenses PomFileHelper.getApacheLicense()
+                        organization PomFileHelper.getLabKeyOrganization()
+                        scm {
+                            connection = 'scm:git:https://github.com/LabKey/testAutomation'
+                            developerConnection = 'scm:git:https://github.com/LabKey/testAutomation'
+                            url = 'scm:git:https://github.com/LabKey/testAutomation'
```

I think the scm `url` property is supposed to be the plain repository url: http://maven.apache.org/pom.html#SCM
```suggestion
                            url = 'https://github.com/LabKey/testAutomation'
```